extensions/amp-subscriptions/0.1/renderer.js

```diff
@@ -65,6 +71,17 @@ export class Renderer {
     });
   }
 
+  addLoadingBar() {
+    if (!this.getRootElement_().querySelector('[subscriptions-section=loading]')) {
```

Comment:
You first need to wait for the full body to be parsed. E.g. `ampdoc.whenReady()`. This service may be created before it's created.

Reply:
Done